VISUALIZERS/DATA_STRUCTURE/ARRAY_VIEW/ARRAY_VIEW.py

```diff
@@ -0,0 +1,107 @@
+from flojoy import flojoy, DataContainer
+import plotly.graph_objects as go
+import numpy as np
+import pandas as pd
+
+CELL_SIZE = 50
+FONT_SIZE = 10
+MAX_ALLOWED_SHAPE = 10
+MIN_ALLOWED_SHAPE = 4
+v_dot = "$\\vdots$"
+d_dot = "$\\ddots$"
+l_dot = "$\\ldots$"
+
+
+def numpy_array_as_table(arr: np.ndarray, placeholder: str):
+    if arr.size > MAX_ALLOWED_SHAPE:
+        converted_type = arr.astype(object)
+        new_arr = converted_type[:MAX_ALLOWED_SHAPE]
+        new_arr[MAX_ALLOWED_SHAPE - 2] = l_dot
+    elif arr.size < MIN_ALLOWED_SHAPE:
+        new_arr = np.full((MIN_ALLOWED_SHAPE,), placeholder, dtype=object)
+        new_arr[: arr.size] = arr[: arr.size]
+    else:
+        new_arr = arr
+    return new_arr.reshape(-1, 1)
+
+
+@flojoy
+def ARRAY_VIEW(dc_inputs: list[DataContainer], params: dict) -> DataContainer:
+    """
+    It takes "ordered_pair", "dataframe", "matrix", and "image" as input type
+    and displays its visualization in array format.
+
+    Parameters
+    ----------
+    None
+
+    Returns
+    -------
+    plotly
+        Visualization of the input data in array format
+    """
+
+    dc_input = dc_inputs[0]
+    match dc_input.type:
+        case "ordered_pair":
+            data = dc_input.y
+            cell_values = numpy_array_as_table(data, l_dot)
+        case "dataframe":
+            data = pd.DataFrame(dc_input.m).to_numpy(dtype=object)
+            data = data[:, :-1]
+            cell_values = numpy_array_as_table(data, l_dot)
+        case "matrix":
+            data = dc_input.m
+            cell_values = numpy_array_as_table(data, l_dot)
+        case "image":
+            red = dc_input.r
+            green = dc_input.g
+            blue = dc_input.b
+
+            if dc_input.a == None:
+                merge = np.stack((red, green, blue), axis=2)
+                merge = merge.reshape(-1, merge.shape[-1])
```

Comment:
Repeating 2 lines in both parts of the if statements can be moved out

Reply:
@JeffDotPng, One of them, includes alpha, and the other doesn't. Do you still think I should move them out?

Reply:
I'm talking about the lines `merge = merge.reshape(-1, merge.shape[-1])` and `cell_values = numpy_array_as_table(merge, l_dot)`